src/main/java/uk/gov/hmcts/reform/translate/ApplicationParams.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.translate;
+
+import org.springframework.beans.factory.annotation.Value;
+
+import java.util.List;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+@Named
+@Singleton
```

Comment:
Is this needed?  Wouldn't Spring only make a single instance of this?

Reply:
hehe you know what I always thought about it . All other projects in ccd using the ApplicationParams like that.  I copied and paste it .  Let me see if we can remove it.

Reply:
NB: changes to address this comment are in PR #44.